site/public/ts/controllers/MainDesignerController.ts

```diff
@@ -30,6 +30,7 @@ import {ANDGate}  from "../models/ioobjects/gates/ANDGate";
 import {ORGate}  from "../models/ioobjects/gates/ORGate";
```

Comment:
Remove changes to MainDesignerController before pushing please 🙏

Reply:
done